pytorch_lightning/utilities/imports.py

```diff
@@ -68,7 +68,9 @@ def _compare_version(package: str, op, version) -> bool:
 _TORCH_GREATER_EQUAL_1_8 = _compare_version("torch", operator.ge, "1.8.0")
 _TORCH_GREATER_EQUAL_1_8_1 = _compare_version("torch", operator.ge, "1.8.1")
 _TORCH_GREATER_EQUAL_1_9 = _compare_version("torch", operator.ge, "1.9.0")
-_TORCH_GREATER_EQUAL_1_10 = _compare_version("torch", operator.ge, "1.10.0")
+_TORCH_GREATER_EQUAL_1_10 = _compare_version("torch", operator.ge, "1.10.0") or _compare_version(
+    "torch", operator.ge, "1.10.0dev"
+)  # todo (sean): remove dev check once 1.10 released
```

Comment:
Perhaps you should use a separate check similar to this one:

https://github.com/PyTorchLightning/pytorch-lightning/commit/3d5e71a7675c024560128f747c70c727fe95abd5#diff-37b14b9ef2f763796ae4fc2762c4ab6018bae62ea90f556aa960d7bc59695d9aR88

Reply:
great idea, will do!